src/runtime_src/core/common/sysinfo.h

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright (C) 2023 Advanced Micro Devices, Inc. All rights reserved.
+#ifndef _xrt_core_common_sysinfo_h_
+#define _xrt_core_common_sysinfo_h_
+
+// Local - Include Files
+#include "config.h"
+
+// 3rd Party Library - Include Files
+#include <boost/property_tree/ptree.hpp>
```

Comment:
You may want to include ptree_fwd.hpp and then ptree.hpp only where the definition is needed.
Obviously all clients of the APIs in sysinfo.h will need ptree.hpp, but that doesn't mean we should include it for them, e.g. relying on indirect includes is not great, so someone like utils.cpp should include ptree.hpp because it needs the definition.